catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/experimental/Extracted.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.types.experimental;
+
+
+public interface Extracted {
+    /**
+     * {@link ddf.catalog.data.Attribute} name for accessing the extracted text of the {@link ddf.catalog.data.Metacard}. <br/>
+     * This attribute will hold the beginning bytes of a document parsed by the TikaInputTransformer or PdfInputTransformer.
```

Comment:
✏️ you can remove the references to specific transformers.  Not really relevant to this attribute and the list could change in the future.